homeassistant/components/radio_browser/media_source.py

```diff
@@ -185,7 +185,7 @@ async def _async_build_by_language(
             return [
                 BrowseMediaSource(
                     domain=DOMAIN,
-                    identifier=f"language/{language.code}",
+                    identifier=f"language/{language.name.lower()}",
```

Comment:
`python-radios` for some reason title-cases all the language names. Luckily we can get away with this because there are no uppercase letters in any of the language names returned by the radio-browser api.

But that would ideally be removed. 

There are also a few broken links where the language name contains a slash, and that is not escaped, but I guess that should be fixed in the library. But that is pretty rare.